src/components/about/About.js

```diff
@@ -96,7 +95,7 @@ const About = () => {
               believe American leadership is essential in the global struggle
               for human rights, so we press the U.S. government and private
               companies to respect human rights and the rule of law. When they
-              fail, we step in to demand reform, accountability and justice.
+              fail, we step in to demand reform, accountability, and justice.
```

Comment:
Thank you for including an Oxford comma. I know that they're considered "optional" in conventional, American English, but I still find them very useful.